src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -152,7 +152,9 @@ class WorkspaceReimburseView extends React.Component {
                     ]}
                 >
                     <View style={[styles.mv3, styles.flexRow, styles.flexWrap]}>
-                        <Text>
+                        {/* Workaround to fix https://github.com/Expensify/App/issues/17368.
+                            TODO: Remove `numberOfLines` once https://github.com/facebook/react-native/pull/35703 is merged and applied to our repo */}
+                        <Text numberOfLines={100}>
```

Comment:
Why use 100 here? In your proposal you said that 5 fixed the issue.

Reply:
Please check https://github.com/Expensify/App/issues/17368#issuecomment-1583959283

Reply:
Ah, ok - I read that as a random number you gave as an example - not the value that you'd be using in your PR, but either works really. As you said, better to use a larger number to be safe.